integreat_cms/cms/views/pages/page_actions.py

```diff
@@ -928,3 +929,70 @@ def render_mirrored_page_field(request, region_slug):
             "page_form": page_form,
         },
     )
+
+
+@login_required
+@region_permission_required
+@permission_required("cms.view_page")
+# pylint: disable=unused-argument
+def get_page_children_ajax(
+    request, region_slug, language_slug, tree_id, lft, rgt, depth
+):
+    """
+    Retrieve the rendered subtree of a given parent page
+
+    :param request: The current request
+    :type request: ~django.http.HttpResponse
+
+    :param region_slug: The slug of the current region
+    :type region_slug: str
+
+    :param language_slug: The slug of the current language
+    :type language_slug: str
+
+    :param tree_id: The :attr:`~integreat_cms.cms.models.pages.page.Page.tree_id` of the parent page
+    :type tree_id: int
+
+    :param lft: The :attr:`~integreat_cms.cms.models.pages.page.Page.lft` field of the parent page
+    :type lft: int
+
+    :param rgt: The :attr:`~integreat_cms.cms.models.pages.page.Page.rgt` field of the parent page
+    :type rgt: int
+
+    :param depth: The :attr:`~integreat_cms.cms.models.pages.page.Page.depth` of the parent page.
+                  A root page has a depth of 1.
+    :type depth: int
+
+    :return: The rendered mirrored page field
+    :rtype: ~django.template.response.TemplateResponse
+    """
+    depth = int(depth)
+    region = request.region
+    language = region.get_language_or_404(language_slug, only_active=True)
+    # List of all ancestors, the requested parent and all direct children
+    pages = region.pages.filter(
+        # This condition queries the parent and all direct children
+        Q(tree_id=tree_id, lft__range=(lft, int(rgt) - 1), depth__lte=depth + 1)
+        # This condition queries all ancestors
+        | Q(tree_id=tree_id, lft__lt=lft, rgt__gt=rgt)
+    ).cache_tree()[0]
+    # If the depth is 1, the first element must be the direct parent, etc.
+    parent = pages[depth - 1]
+    # The remaining pages are the direct children
+    children = pages[depth:]
+    ancestors = [ancestor.id for ancestor in pages[:depth]]
```

Comment:
```suggestion
    # For every depth level, exactly one ancestor is in the list
    ancestor_ids = [ancestor.id for ancestor in pages[:depth]]
```
I think it's better to make it clear what this list only contains ids and not the objects itself.